<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AirportsInspector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QComboBox" name="cb_directions"/>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="pb_flights">
      <property name="text">
       <string>Рейсы</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="l_direction">
      <property name="text">
       <string>Направление:</string>
      </property>
      <property name="buddy">
       <cstring>cb_directions</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="9">
     <widget class="QTableView" name="tv_flights"/>
    </item>
    <item row="8" column="3">
     <widget class="QCalendarWidget" name="cw_date">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QComboBox" name="cb_airports"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="l_connection">
        <property name="text">
         <string>Подключение:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="l_connStatus">
        <property name="text">
         <string>нет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="l_date">
      <property name="text">
       <string>Дата:</string>
      </property>
      <property name="buddy">
       <cstring>cw_date</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QPushButton" name="pb_statistics">
      <property name="text">
       <string>Статистика</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="l_airport">
      <property name="text">
       <string>Аэропорт:</string>
      </property>
      <property name="buddy">
       <cstring>cb_airports</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
